### DISCUSSION TITLE
Adding users to the `pytest-bdd-developers` group

### DISCUSSION THREAD

**Discussion by youtux** [pytest-dev/pytest-bdd#724] [General]
@pytest-dev/core I'm trying to add a user to the `pytest-bdd-developers` group, but I see I don't have permissions. Could you grant me permission to, since I'm admin of `pytest-bdd`?
Same thing for `pytest-factoryboy`.

Thank you.

**Comment by nicoddemus** [comment]
Hi @youtux,

Strange, seems to me you should be able to do this already:

![image](https://github.com/user-attachments/assets/1d19c1a4-6b06-4047-a183-7c733f358724)

![image](https://github.com/user-attachments/assets/a0f25d5b-91e0-4ca6-90c7-5be37367953b)

`pytest-factoryboy` is configured the same way... 🤔 

Any suggestions?

**Reply by youtux** [reply]
Here's how the UI looks to me. I'm missing the "Add a member" button:
<img width="1431" alt="Screenshot 2024-10-19 at 17 57 36" src="https://github.com/user-attachments/assets/7b6ae8a6-c401-4765-8917-41991d3d18e3">

**Reply by nicoddemus** [reply]
That's weird, as Admin you definitely should be able to add new collaborators:

![image](https://github.com/user-attachments/assets/ef5e8b30-195b-45ac-91cb-0d9df8a35ce6)

What do you see if you go to https://github.com/pytest-dev/pytest-bdd/settings/access?

**Reply by nicoddemus** [reply]
(Meanwhile if you are in a rush, post the username you would like to add and I can do it for you)

**Reply by youtux** [reply]
It looks I can indeed add individual users to the project, just not to that group in particular:
![image](https://github.com/user-attachments/assets/1e423de2-df27-4a31-9375-ca582a313086)

**Reply by nicoddemus** [reply]
Hmmm I'm sorry about that, not sure if there's a configuration knob we can tweak for that. Would you like me to add the user myself?

**Reply by youtux** [reply]
Maybe only organization owners can modify the team perms?

**Reply by nicoddemus** [reply]
I added you as a direct admin to `pytest-bdd`, please check if you can now edit the teams.

**Reply by webknjaz** [reply]
@nicoddemus in teams, there's a "Maintainer" role for the team. You can mark somebody that and they should be able to add new members (that are already in the org). That's not really connected to the repo access roles and can be confusing in the UI until you get used to it..

**Reply by nicoddemus** [reply]
Ahh  thanks @webknjaz, found it:

![image](https://github.com/user-attachments/assets/6c6c366a-bd10-45a6-ade9-ca5b366b0528)

@youtux can you test that please?

**Reply by youtux** [reply]
Not solved I'm afraid:
![image](https://github.com/user-attachments/assets/7078cc29-7329-4dc5-a2ff-dfb213decd35)

**Reply by nicoddemus** [reply]
Oh the previous screenshot was from `pytest-bdd-admin`, try now:

![image](https://github.com/user-attachments/assets/4cf54ea2-b7c2-4ef4-8d3b-66d542d1b09e)

**Reply by youtux** [reply]
Ah yes I can manage these groups now, thank you!

Could you add @jsa34 to the pytest-dev org? Otherwise it won't let me add him to the group

**Reply by nicoddemus** [reply]
Sure, invited:

![image](https://github.com/user-attachments/assets/e684d066-3253-4fe7-a6b3-e0cc072fdbea)

**Reply by youtux** [reply]
Nice thx

**Comment by jsa34** [comment]
Thank you @nicoddemus @webknjaz and @youtux 
❤️
